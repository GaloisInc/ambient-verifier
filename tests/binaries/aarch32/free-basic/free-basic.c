#include <stdlib.h>

int main(void) {
  int *x = malloc(sizeof(int));
  int *y = malloc(sizeof(int));
  int *z = malloc(sizeof(int));

  free(x);
  free(y);
  free(z);

  return 0;
}
