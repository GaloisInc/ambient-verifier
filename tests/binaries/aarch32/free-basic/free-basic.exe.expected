successful: 0
failed: 0
